static/stripe.html

```diff
@@ -0,0 +1,646 @@
+          href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css"/>
+    <link rel="stylesheet" type="text/css" href="/min/style-min.css"/>
+
+    <style>
+        .error {
+            color: #F44336;
+            padding-left: 8px;
+        }
+
+        /* Initial page load animation */
+        .fade-in {
+            opacity: 1;
+            -webkit-transition: opacity .33s ease;
+            transition: opacity .33s ease;
+        }
+        .invisible {
+            opacity: 0!important;
+        }
+
+        /* Hide the browser's up/down buttons on <input type=number> */
+        input::-webkit-outer-spin-button, input::-webkit-inner-spin-button {
+            -webkit-appearance: none;
+            margin: 0;
+        }
+        input[type=number] {
+            -moz-appearance: textfield;
+        }
+    </style>
+</head>
+<body>
+<header>
+    <div class="navbar-fixed">
+        <nav role="navigation">
+            <div class="nav-wrapper container">
+                <a href="/" class="brand-logo"><h1>Impact</h1></a>
+            </div>
+        </nav>
+    </div>
+</header>
+<div id="initial-load" class="invisible fade-in">
+    <div class="container">
+        <div class="row">
+            <p class="col s12">
+                Donations of any size are appreciated! <strong>The minimum donation to receive premium features is <span class="required-amount"></span> .</strong>
+                <br/>
+                This donation form is currently in early beta-testing, please report any issues you find to
+                <a  target="_blank" href="https://github.com/ImpactDevelopment/ImpactServer/issues">this bug tracker</a>.
+                Please avoid including any private information (including payment IDs, emails, etc) when reporting issues on the bug tracker.
+                If private information is needed, please message <code>LeafHacker#5373</code> privately on <a target="_blank" href="/discord">Discord</a>!
+            </p>
+        </div>
+    </div>
+    <form id="amount-form" class="container">
+        <div class="row">
+            <div class="input-field col s3 m2">
+                <select name="currency" id="currency" required tabindex="0">
+                    <option value="" disabled selected>Choose...</option>
+                </select>
+                <label for="currency">Currency</label>
+                <span class="helper-text" data-error="Invalid currency" data-success="">The currency to use</span>
+            </div>
+            <div class="input-field col s9 m3 l2">
+                <label for="amount" class="active">Amount</label>
+                <input id="amount" name="amount" type="number" data-type="currency" min="0.5" tabindex="1" required autofocus />
+                <span class="helper-text" data-error="Invalid amount" data-success="">The amount to donate</span>
+            </div>
+            <div class="input-field col s12 m5 l6">
```

Comment:
not sure where exactly i should mention it so im doing it in a comment near the affected field
when you use the browsers autofill (or at least firefox idk) it will complain about the address being invalid until you manually input some text or click around the site a bit
![image](https://user-images.githubusercontent.com/24397680/104151757-cdab4a00-53ab-11eb-85d9-2828a2bcb335.png)
message showing its affected at least one user in testing:
![image](https://user-images.githubusercontent.com/24397680/104151876-17943000-53ac-11eb-899e-cce745a0028c.png)